### PR TITLE
Add repeated snapshot failure troubleshooting guide

### PR DESCRIPTION
**Pull request by jbaiera**
This troubleshooting guide is what will be returned from the SLM health indicator when a SLM policy has suffered from too many repeat failures without a successful execution.

Snapshots can fail for a multitude of reasons, which makes it difficult to give a full guide on all steps that are needed to fix them. We might be able to extend this guide in the future if we notice a trend in snapshot failures from end users.

### CODE REVIEW

File: docs/reference/tab-widgets/troubleshooting/snapshot/repeated-snapshot-failures.asciidoc [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+}
+----
+// TESTRESPONSE[skip:the result is for illustrating purposes only]
+<1> The affected snapshot lifecycle policy.
+<2> The information about the last failure for the policy.
+<3> The time when the failure occurred in millis. Use the `human=true` request parameter to see a formatted timestamp.
+<4> Error details containing the reason for the snapshot failure.
++
+
+Snapshots can fail for a variety reasons. If the failures are due to configuration errors, consult the
```
Comment (andreidan) [inline docs/reference/tab-widgets/troubleshooting/snapshot/repeated-snapshot-failures.asciidoc:86]: Shall we link to https://www.elastic.co/guide/en/cloud-enterprise/current/ece-manage-repositories.html to help correct the configuration errors ? 

Also one of the most common causes will be multiple deployments writing to the same repo for which we have a troubleshooting guide. Shall we mention and link to it? [ela.st/fix-repository-integrity](http://ela.st/fix-repository-integrity)

File: docs/reference/tab-widgets/troubleshooting/snapshot/repeated-snapshot-failures.asciidoc [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+documentation for the repository that the automated snapshots are using. In the event that snapshots are failing for
+other reasons check the logs on the elected master node during the snapshot execution period for more information.
```
Comment (andreidan) [inline docs/reference/tab-widgets/troubleshooting/snapshot/repeated-snapshot-failures.asciidoc:88]: I wonder if we should recommend contacting support as opposed to checking logs? @Leaf-Lin what do you think?

**Review by andreidan — COMMENTED** [review]
Thanks for adding this Jimmy ❤️ 

LGTM on my side with a few questions (also requesting a review from @debadair too)

File: docs/reference/tab-widgets/troubleshooting/snapshot/repeated-snapshot-failures.asciidoc (hunk 2 above)
Comment (Leaf-Lin) [reply docs/reference/tab-widgets/troubleshooting/snapshot/repeated-snapshot-failures.asciidoc:88]: I prefer providing the instruction to check the logs, customers can always contact support if they still have a problem, but checking the logs gives them some action to potentially rectify the issue if the logs contain sufficient details.

**Review by andreidan — APPROVED** [review]
LGTM, thanks for working on this Jimmy 🚀

**Pull request merged**